Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/Dockerfile-torchserve

```
@@ -21,9 +21,7 @@ RUN mkdir -p ${MLFLOW_HOME}/scripts && \
     rm -rf /var/lib/apt/lists/* && \
     git clone https://github.com/pytorch/serve.git && \
     cd serve && python ./ts_scripts/install_dependencies.py && \
-    pip install --no-cache-dir torch==1.7.1+cpu torchvision==0.8.2+cpu torchaudio==0.7.2 torchserve torch-model-archiver -f https://download.pytorch.org/whl/torch_stable.html && \
-    pip install --no-cache-dir -r /tmp/requirements.txt \
-    pip install --no-cache-dir -r /tmp/requirements-torchserve.txt
+    pip install --no-cache-dir -r /tmp/requirements.txt
```

Maybe we should use the official repo.
```suggestion
    pip install --no-cache-dir -r /tmp/requirements.txt -f https://download.pytorch.org/whl/torch_stable.html
```

---

This will be solved by having a base image